test/support/test_backend.ex

```diff
@@ -9,7 +9,7 @@ defmodule VintageNetWizard.Test.Backend do
   @impl true
   def access_points(_) do
     [
-      %{
+      %VintageNet.WiFi.AccessPoint{
```

Comment:
Tests are failing because you changed this to the struct and `Jason` decoder doesn't know how to derive the fields into json.

Reply:
Looks like we're getting around that by converting all access points to maps when we get them from `VintageNet`:
https://github.com/nerves-networking/vintage_net_wizard/blob/a3188c246d8f4f8db80dfa5e8050ba360d9047e4/lib/vintage_net_wizard/backend/default.ex#L133

But, that does go against the spec which says its a list of `AccessPoint.t()`.

So probably better to just remove it here, make the necessary changes to VintageNet to derive the structs, then finish adjusting here the correct way

Reply:
Agreed. I pushed an update.

Reply:
nvm..you changed it all _right_ as I commented all this..

ignore me

![](http://giphygifs.s3.amazonaws.com/media/4pMX5rJ4PYAEM/200.gif)